bin/hdfcoinc/pycbc_multiifo_coinc_statmap

```diff
@@ -323,68 +317,60 @@ while numpy.any(louder_foreground == 0):
 
     # Find the index of the loud foreground trigger to remove next. And find
     # the index in the list of original foreground triggers.
-
     rm_trig_idx = numpy.where(all_trigs.stat[:] == fore_stat[max_stat_idx])[0][0]
     orig_fore_idx = numpy.where(orig_fore_stat == fore_stat[max_stat_idx])[0][0]
 
     # Store any foreground trigger's information that we want to
     # hierarchically remove.
-
     f['foreground/ifar'][orig_fore_idx] = conv.sec_to_year(ifar[max_stat_idx])
     f['foreground/fap'][orig_fore_idx] = fap[max_stat_idx]
 
     logging.info("Removing foreground trigger that is louder than the inclusive background.")
 
     # Remove the foreground trigger and all of the background triggers that
     # are associated with it.
-
     ave_rm_time = 0
     for ifo in args.ifos:
         ave_rm_time += all_trigs.data['%s/time' % ifo][rm_trig_idx] / len(args.ifos)
 
     ind_to_rm = {}
-
     for ifo in args.ifos:
         ind_to_rm[ifo] = veto.indices_within_times(all_trigs.data['%s/time' % ifo],
                               [ave_rm_time - args.hierarchical_removal_window],
                               [ave_rm_time + args.hierarchical_removal_window])
-
     indices_to_rm = []
     for ifo in args.ifos:
         indices_to_rm = numpy.concatenate([indices_to_rm, ind_to_rm[ifo]])
-
     all_trigs = all_trigs.remove(indices_to_rm)
 
-    fore_locs = all_trigs.timeslide_id == 0
-    # The foreground trigger has been removed, continue with typical statmap operations.
+    # NB the values calculated below are not used!
```

Comment:
Why not remove these lines if we aren't going to use them?

Reply:
I don't know if it's worth thinking about whether we might use them .. I guess the exclusive background is what it is independent of any hierarchical removal, so indeed they were never useful.